minecraft_servers/nidocraft/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+    "server_name": "nidocraft",
+    "nice_name": "Nidocraft Network",
+    "direct_ip": "play.nidocraft.net",
+    "supported_languages": [
+       "en"
+    ],
+    "social": {
+      "web_shop": "https://store.nidocraft.net/",
+      "tiktok": "nidotok",
+      "instagram": "nidocraftnetwork",
+      "discord": "https://discord.gg/WvCH6NKXBD",
+      "youtube": "https://www.youtube.com/c/NidocraftNetwork"
+    },
+    "gamemodes": {
+      "ffa": {
+        "name": "Free For All", 
+        "color": "#00FF00"
+      },
+      "bridges": {
+        "name": "Bridges",
+        "color": "#ED4C4C"
+      }
+    },
+    "discord": {
+      "server_id": 402362480753377280,
+      "rename_to_minecraft_name": true
```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.